appveyor.yml

```diff
@@ -1,22 +1,18 @@
 environment:
-  LEATHERMAN_VERSION: 1.1.0
+  LEATHERMAN_VERSION: 1.1.1
 
 install:
-  - choco install -y mingw-w64 -Version 4.8.3 -source https://www.myget.org/F/puppetlabs
+  - choco install -y mingw-w64 -Version 5.2.0 -source https://www.myget.org/F/puppetlabs
   - choco install -y cmake -Version 3.2.2 -source https://www.myget.org/F/puppetlabs
+  - choco install -y gettext -Version 0.19.6 -source https://www.myget.org/F/puppetlabs
+  - choco install -y pl-toolchain-x64 -Version 2015.12.01.1 -source https://www.myget.org/F/puppetlabs
```

Comment:
The pl-toolchain is specifically https://github.com/puppetlabs/pl-build-tools-vanagon/blob/master/files/cmake/windows-x64-toolchain.cmake.